src/use-swr.ts

```diff
@@ -366,7 +370,7 @@ function useSWR<Data = any, Error = any>(
     // and trigger a revalidation
 
     const currentHookData = stateRef.current.data
-    const latestKeyedData = cache.get(key) || config.initialData
+    const latestKeyedData = cache.get(key)
```

Comment:
This should fallback to `config.initialData` as before for the case where SWR is running server-side